### PR TITLE
added coveralls

### PR DESCRIPTION
**Pull request by yuce**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # Python Client for Pilosa
 
 <a href="https://github.com/pilosa"><img src="https://img.shields.io/badge/pilosa-v0.3.1-blue.svg"></a>
-<a href="https://travis-ci.com/pilosa/python-pilosa"><img src="https://travis-ci.com/pilosa/python-pilosa.svg?token=Peb4jvQ3kLbjUEhpU5aR&branch=master"></a>
 <a href="https://pypi.python.org/pypi/pilosa"><img src="https://img.shields.io/pypi/v/pilosa.svg?maxAge=2592000"></a>
 <a href="http://pilosa.readthedocs.io/en/latest/?badge=latest"><img src="https://img.shields.io/badge/docs-latest-brightgreen.svg?style=flat"></A>
+<a href="https://travis-ci.com/pilosa/python-pilosa"><img src="https://travis-ci.com/pilosa/python-pilosa.svg?token=Peb4jvQ3kLbjUEhpU5aR&branch=master"></a>
+<a href="https://coveralls.io/github/pilosa/python-pilosa?branch=coveralls"><img src="https://coveralls.io/repos/github/pilosa/python-pilosa/badge.svg?branch=coveralls"></a>
```
Comment (codysoyland) [inline README.md]: Should branch=master when we merge this?

Comment (yuce) [inline README.md]: Thanks, missed that. Updated.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
 # Python Client for Pilosa
 
 <a href="https://github.com/pilosa"><img src="https://img.shields.io/badge/pilosa-v0.3.1-blue.svg"></a>
-<a href="https://travis-ci.com/pilosa/python-pilosa"><img src="https://travis-ci.com/pilosa/python-pilosa.svg?token=Peb4jvQ3kLbjUEhpU5aR&branch=master"></a>
 <a href="https://pypi.python.org/pypi/pilosa"><img src="https://img.shields.io/pypi/v/pilosa.svg?maxAge=2592000"></a>
 <a href="http://pilosa.readthedocs.io/en/latest/?badge=latest"><img src="https://img.shields.io/badge/docs-latest-brightgreen.svg?style=flat"></A>
+<a href="https://travis-ci.com/pilosa/python-pilosa"><img src="https://travis-ci.com/pilosa/python-pilosa.svg?token=Peb4jvQ3kLbjUEhpU5aR&branch=master"></a>
```
Comment (codysoyland) [inline README.md]: Can you set this up with the open source Travis CI? (travis-ci.org). This one isn't visible by the general public.

Comment (yuce) [inline README.md]: @codysoyland up.

**Pull request merged**